packages/backend-for-frontend/src/services/clientService.ts

```diff
@@ -249,6 +250,8 @@ export function clientServiceBuilder(
           )
         )
       );
+
+      return name ? users.filter((user) => user.name === name) : users;
```

Comment:
a couple of doubts 🤔 
1 - the "name" can be use like a DB query filter instead of filter for overall query result? In this way we prevent unnecessary workload on DB.
2 - the "name" query param must be exactly equal to user name or it can be a substring? Have you considered a partial name match in your use cases?

Reply:
🚫  `name` is not valid query param for API in selfcareV2ClientApi.yml .